.eslintrc.json

```diff
@@ -1,19 +1,21 @@
 {
-  "plugins": [
-    "@typescript-eslint", // mandatory typescript plugin
-    "sonarjs" // sonar plugin for code analysis
-  ],
-  "extends": [
-    "airbnb-base", // use instead on eslint recommended as a stricter alternative
-    "airbnb-typescript/base", // airbnb rules overrides for typescript
-    "plugin:@typescript-eslint/recommended",
-    "plugin:sonarjs/recommended", // sonar
-    "prettier" // compatibility with prettier (disable conflicting rules) NB: this is the `eslint-config-prettier` it should not be duplicated
-  ],
   "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint", "import"],
   "parserOptions": {
-    "ecmaVersion": "latest",
-    "project": "./tsconfig.json" // Specify the path to your tsconfig.json file
+    "project": "./tsconfig.json",
+    "ecmaVersion": "latest"
   },
-  "rules": {}
+  "extends": [
+    "airbnb-typescript/base",
+    "plugin:@typescript-eslint/recommended",
+    "prettier"
+  ],
+  "rules": {
+    "import/no-extraneous-dependencies": [
+      "error",
+      {
+        "devDependencies": true
+      }
+    ]
```

Comment:
`import/no-extraneous-dependencies` accepting `devDependencies` should apply only to the `scripts`and `test` files

Reply:
I can fix it now but ` @nomicfoundation/hardhat-toolbox`  & `hardhat` should be set as `dependency` and not `dev-dependency`

Reply:
Commit: 7f264c6b53878c50d596cf2684b64d4c2a102e88
Fix